### PR TITLE
Address slf4j warnings

### PR DESCRIPTION
**Pull request by jethronap**
pls consider this PR in order to close corresponding issue.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -48,14 +48,36 @@
...
+
+        <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-api -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.0-alpha0</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-simple -->
+        <dependency>
```
Comment (pockerman) [inline pom.xml:76]: so this is the only change required???

**Review by pockerman — CHANGES_REQUESTED** [review]
If you merge this branch into the master...all items marked as green will be merged also :) :) However you don't want that....

File: pom.xml (hunk 1 above)
Comment (jethronap) [reply pom.xml:76]: yes this gets rid of the warning.

**Comment by jethronap** [comment]
resolved conflicts I think we're good to go.

**Review by pockerman — CHANGES_REQUESTED** [review]
I think we sould delete this branch altogether. We simply add the ```slf4j``` in ```pom.xml``` and yet there are all these changes.  If we simply branch off from the current master and do the change things will be simpler. What do you think?

**Comment by jethronap** [comment]
ok I'll put these changes in create_visualization_classes. in its pom.xml and then I'll delete this branch

**Pull request closed without merging**